### ISSUE TITLE
phik cannot install on arm

### ISSUE THREAD

**Issue by Wh1isper** [KaveIO/PhiK#48]
related to https://github.com/ydataai/pandas-profiling/issues/943

It seems like sdist broke, in phik-0.12.1.tar.gz there is no setup.pu, this is unusual

`pip install --upgrade phik`

log:

Collecting phik
  Downloading http://myownrepo/repository/pypi/packages/phik/0.12.1/phik-0.12.1.tar.gz (600 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 600.8/600.8 KB 42.9 MB/s eta 0:00:00
ERROR: phik from http://myownrepo/repository/pypi/packages/phik/0.12.1/phik-0.12.1.tar.gz#sha256=63cf160c8950ec46da7a33165deef57f27d29f24b83cf4dd028aa0cb97b73af6 does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.

**Comment by RUrlus** [comment]
@Wh1isper Would you be willing to test an MacOS arm wheel?

We have just added support for `arm64` wheels, #49, but we don't have access to Apple silicone to test the wheel.

If so, you can download the set of wheels [here](https://github.com/KaveIO/PhiK/suites/5737462685/artifacts/190176170) and install the Python 3.10 wheel using:
```bash
pip install <path_to_wheel_dir>/phik-0.12.1-cp310-cp310-macosx_11_0_arm64.whl
```
Than run the tests using:

```bash
git clone https://github.com/KaveIO/PhiK.git
pytest PhiK/tests/phik_python/test_phik.py
```

**Comment by Wh1isper** [comment]
@RUrlus 
Sorry I don't have a MacOS arm machine, I tested on my linux arm machine using the new `phik-0.12.1.tar.gz` installation and everything works fine

`pip install phik-0.12.1.tar.gz`

**Comment by RUrlus** [comment]
@Wh1isper Ah OK, I assumed you had an MacOS arm machine. Thanks for letting us know it works

**Comment by mbaak** [comment]
Should be fixed with phik==0.12.2

**Comment by Wh1isper** [comment]
Thanks！
